homeassistant/helpers/area_registry.py

```diff
@@ -69,31 +115,33 @@ async def _async_migrate_func(
 class AreaRegistry:
     """Class to hold a registry of areas."""
 
+    areas: AreaRegistryItems
+    _area_data: dict[str, AreaEntry]
+
     def __init__(self, hass: HomeAssistant) -> None:
```

Comment:
Nice change!

It would be nice to have the same `values()` optimization we have in the entity registry here as well since we do have places where areas is enumerated that way
```
homeassistant/components/spc/alarm_control_panel.py:    async_add_entities([SpcAlarm(area=area, api=api) for area in api.areas.values()])
homeassistant/helpers/area_registry.py:        return self.areas.values()
homeassistant/helpers/area_registry.py:            for entry in self.areas.values()
homeassistant/helpers/intent.py:    for maybe_area in areas.areas.values():
```

Reply:
Hmm, but the PR does include that already.
https://github.com/home-assistant/core/blob/5bf98a6e0240a167cf2f704591adb2d8b76d1589/homeassistant/helpers/area_registry.py#L54-L56

Reply:
I'm not sure how I missed that 🤦 

Thanks!